activesupport/lib/active_support/cache.rb

```diff
@@ -644,6 +652,18 @@ def merged_options(call_options)
           end
         end
 
+        # Normalize aliased options to their canonical form
+        def normalize_options(options)
+          return unless options
```

Comment:
This conditional is not needed since you always call this method when option is defined.